Python/bytecodes.c

```diff
@@ -1381,6 +1388,10 @@ dummy_func(
             null = NULL;
         }
 
+        macro(LOAD_GLOBAL_MODULE) = _SKIP_CACHE + _GUARD_GLOBALS_VERSION + _SKIP_CACHE + _LOAD_GLOBAL_MODULE;
```

Comment:
Is this better? (You can also add a comment on each item).

```suggestion
        macro(LOAD_GLOBAL_MODULE) = _SKIP_CACHE +
                                    _GUARD_GLOBALS_VERSION +
                                    _SKIP_CACHE +
                                    _LOAD_GLOBAL_MODULE;
```